### DISCUSSION TITLE
How to turn off LTSP the easiest way?

### DISCUSSION THREAD

**Discussion by ghost** [ltsp/ltsp#260] [Q&A]
I have 2 same Dell computers in the class. On one I have Zorin and it works as an LTSP server, on the other (also Zorin) I work as a teacher. I often want to test different solutions, but the LTSP server must work non-stop. I came up with the idea that I would also install LTSP on a second computer and test other solutions.
The question is simple how to enable and disable the LTSP server so that only one is visible on the network.

**Comment by enaut** [comment]
I usually use:

```
sudo systemctl start dnsmasq
sudo systemctl stop dnsmasq
```

Note that on your testing system you should not start dnsmasq automatically. If you run it at both systems you get undefined behaviour - it could be either or, or even a mixed result. So while testing stop the main ltsp server's dnsmasq and while running make sure your test-dnsmasq is stopped.

**Comment by alkisg** [comment]
You may also use the following to disable/enable the dnsmasq DHCP services. These then remain disabled/enabled even after reboot:

```shell
# Disable Proxy and Real DHCP in dnsmasq:
sudo ltsp dnsmasq -p0 -r0

# Enable Proxy and Real DHCP in dnsmasq:
sudo ltsp dnsmasq
```

**Comment by ghost** [comment]
Thanks, it works great.
One more problem with the IP of the computers on which the students work. Epoptes works without a problem, but I'm testing Veyon and after a long shutdown of the LTSP server, the IP addresses of computers change and I have to set the Veyon configuration again. How to set the student's IP address permanently.

**Comment by alkisg** [comment]
Hello,

in the default LTSP setup, an external router is used to assign the IP addresses.
Is this the case for you?
If so, you should set up static addresses from your router (or other DHCP server).

If your router isn't very configurable, you should at least restrict its range (e.g. from 192.168.1.100 to 192.168.1.254), so that then you have the addresses from .2 to .99 for LTSP clients.

**Comment by ghost** [comment]
This is very bad news :(
The external router assigns the IP addresses and in addition I do not have access to its configuration (external company)

**Comment by enaut** [comment]
Well you can set fixed IP-Adresses to your clients. I think servers should be fixed anyways. You have to be careful, that they do not collide with the automatic ones. Our Router here assigns  < .100 IP-Adresses so I'm free to use everything below .100 for static IP-Adresses. If your router uses  the whole range it usually starts at the beginning. So you could use the IP-Adresses at the end of the range for static adresses.

**Comment by ghost** [comment]
IP addresses are in the range 192.168.104.210 - 230, but this is not a problem. How to assign a fixed IP address to a specific computer using the LTSP server.

**Comment by alkisg** [comment]
This is untested, but it should work. In ltsp.conf, you put:

```shell
[mac:address1]
KERNEL_PARAMETERS="ip=192.168.1.101:\${srv}:\${gateway}:\${netmask}"
```

And then you run `ltsp ipxe`.

**Comment by ghost** [comment]
Did not work :(

**Comment by alkisg** [comment]
Upload your /srv/tftp/ltsp/ltsp.ipxe to see if the parameters made it there.

**Comment by alkisg** [comment]
I just tested the [static IP method](https://github.com/ltsp/community/issues/80#issuecomment-572991699) and it works fine.
